pkg/rtc/transport.go

```diff
@@ -376,6 +366,15 @@ func NewPCTransport(params TransportParams) (*PCTransport, error) {
 			Logger: params.Logger,
 		})
 		t.streamAllocator.Start()
+
+		if params.CongestionControlConfig.UseTWCC {
+			t.sendSideBWE = sendsidebwe.NewSendSideBWE(params.Logger)
+			t.pacer = pacer.NewNoQueue(params.Logger, t.sendSideBWE)
```

Comment:
Using the `NoQueue` pacer (all packets going out from single goroutine) and `PassThrough` pacer for REMB. We can potentially use `NoQueue` for REMB too.